azure-pipelines.yml

```diff
@@ -278,7 +278,7 @@ jobs:
         - win-x64
         - ubuntu-x64
         - win-arm64
-        - ubuntu-arm64
```

Comment:
which machine was `ubuntu-arm64`?

Reply:
It was the Ubuntu Ampere machines before; in this case I just added the -ampere to stay consistent with the windows -ampere.